### ISSUE TITLE
Create FronEnd design skeleton

### ISSUE THREAD

**Issue by SabuhiYusif** [SabuhiYusif/Tasker#39]

**Comment by SabuhiYusif** [comment]
This is the basic Boards Page
![image](https://user-images.githubusercontent.com/15803706/88453548-18c2b680-ce79-11ea-8997-41662614e3b2.png)

**Comment by javidy** [comment]
overall looks good. Can we change colors on the right side? They look a little less fancy.  and how can I get access to .html file?

**Comment by SabuhiYusif** [comment]
Yes we can change that.
Get access to html file ?

**Comment by javidy** [comment]
I meant where did you create this skeleton? is it in front-end github project?

**Comment by SabuhiYusif** [comment]
Yes, pls accept my invitation https://github.com/SabuhiYusif/TaskerFrontEnd
